aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -1813,6 +2034,9 @@ module aptos_framework::delegation_pool {
     #[test_only]
     const COMMISSION_CHANGE_DELEGATION_POOL: u64 = 42;
 
+    #[test_only]
```

Comment:
Also start separating the tests into a separate file under a tests/ directory

Reply:
We will separate tests in bulk on another PR.